partner/WHMCS/modules/servers/jetpack/jetpack.php

```diff
@@ -0,0 +1,365 @@
+<?php
+
+use WHMCS\Database\Capsule;
+
+if (!defined("WHMCS")) {
+    die("This file cannot be accessed directly");
+}
+
+
+/**
+ * A WHMCS module for use by Jetpack hosting partners to provision Jetpack plans.
+ * The module provides functionality for partner hosts to be able to save their
+ * client id and secret to request an access token for provisioning plans.
+ *
+ * Plans available for provisioning include free, personal, premium and professional
+ *
+ * A host has options to either provision(Create == WHMCS equivalent functional term)
+ * or Cancel(Terminate == WHMCS equivalent functional term) from the WHMCS client area.
+ *
+ * Host setup for custom fields is currently required in order to use the module.
+ *
+ */
+
+/**
+ * Jetpack Meta Data for WHMCS module. This is what is shown specifically when the host
+ * is setting up the module.
+ *
+ * @return array
+ */
+function jetpack_MetaData()
+{
+    return [
+        'DisplayName' => 'Jetpack by Automattic',
+        'Description' => 'Use this module to provision Jetpack plans with your Jetpack hosting partner account',
+        'APIVersion' => '1.1',
+        'RequiresServer' => false,
+    ];
+}
+
+
+/**
+ * Basic configuration options required for a partner to get a Jetpack plan provisioned. Currently a partner
+ * client id and secret are the only host partner options needed to get an access token to provision a Jetpack plan
+ * @return array
+ */
+function jetpack_ConfigOptions()
+{
+    return [
+        'Jetpack Partner Client ID' => [
+            'Type' => 'text',
+            'Size' => '256',
+        ],
+        'Jetpack Partner Client Secret' => [
+            'Type' => 'text',
+            'Size' => '256',
+        ]
+    ];
+}
+
+
+/**
+ * Equivalent to /provision. Create a Jetpack plan using
+ * a Jetpack Hosting partner account. WHMCS expects the string "success"
+ * to be returned if the process is completed successfully otherwise
+ * a string can be returned which will be logged as part of the error.
+ *
+ * Pre Provisioning Steps:
+ *  Module requirements are validated and will return an error string
+ *  if the module was not setup correctly. An error string will also be
+ *  returned in the even that a request to provision a new returns a
```

Comment:
"in the even" => "in the event"

Reply:
Fixed this and reviewing all the doc blocks again.